docker/k8s/sgx-enclave-manager-deployment.yaml

```diff
@@ -15,21 +15,21 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: avalon-sgx-enclave-manager
+  name: avalon-enclave-manager
```

Comment:
In K8s both SGX SIM and SGX HW pods can be running at same time. At least I use it usually for developmnet/testing purpose. Else every time we need to delete the deployment of SGX SIM pods before starting SGX HW pods and vice-versa. Are you facing any issue if we name SGX SIM and SGX HW pods for enclave manager differently ?

Reply:
I couldn't get your point. When you say SGX SIM and SGX HW pods, are you referring to enclave manager pod?

Reply:
Yes. But this PR anyways has a service end point for enclave manager. So looks ok. Let's keep it as per this PR for now. If we face any issues later we can change.